wdio.conf.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config();
+
+const path = require('path');
+const { UtamWdioService } = require('wdio-utam-service');
+// use prefix 'DEBUG=true' to run test in debug mode
+const { DEBUG } = process.env;
+const TIMEOUT = DEBUG ? 60 * 1000 * 30 : 60 * 1000;
+
+exports.config = {
+    runner: 'local',
+    specs: ['force-app/test/**/*.spec.js'],
+    maxInstances: 1,
+    capabilities: [
+        {
+            maxInstances: 1,
+            browserName: 'chrome',
+            'goog:chromeOptions': {
+                // to run chrome headless the following flags are required
+                // (see https://developers.google.com/web/updates/2017/04/headless-chrome)
+                // args: ['--headless', '--disable-gpu']
+            }
+        }
+    ],
+    logLevel: 'debug',
+    bail: 0,
+    // timeout for all waitFor commands
+    waitforTimeout: TIMEOUT,
+    connectionRetryTimeout: 120000,
+    connectionRetryCount: 3,
+    automationProtocol: 'webdriver',
+    services: ['chromedriver', [UtamWdioService, { implicitTimeout: 0 }]],
+    framework: 'jasmine',
+    reporters: ['spec'],
+    jasmineNodeOpts: {
```

Comment:
In the WDIO documentation I can only find jasmineOpts, no jasmineNodeOpts. Is this something that may cause issues?
https://github.com/webdriverio/webdriverio/tree/main/packages/wdio-jasmine-framework#jasmineopts-options

Reply:
Same remark as above, this is taken from here:
https://github.com/salesforce/utam-js-recipes/blob/main/wdio.conf.js#L37

@olivier-martin-sf does this have an impact on the Jasmine configuration?

Reply:
I think that [`jasmineNodeOpts`](https://v6.webdriver.io/docs/options.html#mochaopts--jasminenodeopts) is the property name used in WDIO v6 which is the version we use in the `utam-js-recipes` repo. If you are using WDIO v7, the option is indeed named `jasmineOpts`

> Note that we only support WDIO v6 officially. I am about to upgrade the `wdio-utam-service` to make it work with WDIO v7. This is a bit tricky to do as we had to patch some of the WDIO internals to fix their shadow DOM support but this is a long overdue update which hopefully will happen in the next 2 weeks